programmers/두-정수-사이의-합.js

```diff
@@ -0,0 +1,9 @@
+function solution(a, b) {
+  if (a > b) [a, b] = [b, a];
+  var answer = 0;
```

Comment:
밑에서 값이 변하므로 ```let```을 써주는게 맞습니다 :D
anwser보다는 total이 좋겠네요
```suggestion
  let total = 0;
```